quickwit/quickwit-cluster/src/cluster.rs

```diff
@@ -510,35 +539,39 @@ mod tests {
         let transport = ChannelTransport::default();
         let cluster = create_cluster_for_test(Vec::new(), &[], &transport, false).await?;
         let members: Vec<SocketAddr> = cluster
-            .members()
+            .ready_members()
+            .await
             .iter()
             .map(|member| member.gossip_advertise_addr)
             .collect();
         let expected_members = vec![cluster.gossip_listen_addr];
         assert_eq!(members, expected_members);
         assert!(!cluster.is_self_node_ready().await);
-        let cluster_state = cluster.state().await;
-        let self_node_state_not_ready = cluster_state
-            .state
+        assert_eq!(cluster.ready_members_from_chitchat_state().await.len(), 0);
```

Comment:
I suspect clippy will annoy you 
and tell you to make it

assert!(cluster.ready_members_from_chitchat_state().await.is_empty());